Cargo.toml

```diff
@@ -25,8 +25,8 @@ wasm = ["serde_json", "dprint-core/wasm"]
 
 [dependencies]
 dprint-core = { version = "0.31.0", features = ["formatting"] }
-swc_common = "=0.10.2"
-swc_ecmascript = { version = "=0.7.7", features = ["parser"] }
+swc_common = "=0.10.3"
+swc_ecmascript = { version = "=0.8.1", features = ["parser"] }
```

Comment:
Not 0.8.2?

Reply:
Actually gonna wait for another release, closing for now